### DISCUSSION TITLE
有製作的步驟嗎

### DISCUSSION THREAD

**Discussion by rwlin** [WhatTheBlock/WindowsSimplify#23] [General]
想做極度精簡 + rdp 的，請問有製作步驟或教學嗎？

**Comment by WhatTheBlock** [comment]
以前有接觸過製作精簡系統嗎？
建議先學習使用MSMG Toolkit、NTlite、Dism++這幾個工具
再進階一點的就是WinReducer

**Reply by rwlin** [reply]
win7 時代用過 NTLite ...
不知道您是用哪一套做的?

**Reply by WhatTheBlock** [reply]
都有用喔
沒有限定只能用一種
但精簡工具的運作方式略有差異
因此排除問題也是一大課題

通常我會以下列順序來精簡系統
MSMG Toolkit → NTlite → Dism++
如有需要極限精簡則最後在加上WinReducer

**Reply by rwlin** [reply]
謝謝您
